doc/chapters/api/index.rst

```diff
@@ -24,88 +13,276 @@ Methods
 
 Ping
 ----
-Ping()
+Controls availability of com.pelagicore.SoftwareContainerAgent interface
+
+:Parameters:
+        - **None**
+
+:Return Value:
+        - **None** 
+
+:Example Usage:
+
+::
+
+        dbus-send --system --print-reply \
+        --dest=com.pelagicore.SoftwareContainerAgent \
+        /com/pelagicore/SoftwareContainerAgent \
+        com.pelagicore.SoftwareContainerAgent.Ping
 
 CreateContainer
 ---------------
-:containerIDPrefix: string
-:config: string
+Creates a new container and returns created container id.
+
+:Parameters:
+        - **IDPrefix** : ``string`` prefix for random container id.
+        - **config** : ``string`` config file in json format.
+        
+                Example config JSON::
+        
+                [{"enableWriteBuffer": true}]
+
+ 
```

Comment:
It is indeed but there are no blank lines in the html. For code snippet reStructuredText uses this structure.